src/backend/aspen/api/schemas/phylo_runs.py

```diff
@@ -2,28 +2,34 @@
 import re
 from typing import Dict, List, Optional
 
-from pydantic import constr, Field, root_validator, StrictStr, validator
+from pydantic import conint, constr, Field, root_validator, StrictStr, validator
 
 from aspen.api.schemas.base import BaseRequest, BaseResponse
 from aspen.database.models import TreeType
 
 # What kinds of ondemand nextstrain builds do we support?
-PHYLO_TREE_TYPES = {
-    TreeType.NON_CONTEXTUALIZED.value: "non_contextualized.yaml",
-    TreeType.TARGETED.value: "targeted.yaml",
-}
+PHYLO_TREE_TYPES = [
+    TreeType.OVERVIEW.value,
+    TreeType.NON_CONTEXTUALIZED.value,
+    TreeType.TARGETED.value,
+]
```

Comment:
We don't need to configure YAML filenames here anymore, but we *do* need to support creating overview trees via API calls now.